README.md

```diff
@@ -1,13 +1,56 @@
-# Overview
+[![Build Status](https://travis-ci.org/googleapis/gax-dotnet.svg?branch=master)](https://travis-ci.org/googleapis/gax-dotnet)
+
+Google API Extensions for .NET
+===
+
 Google API Extensions for .NET (`Google.Api.Gax`) is a set of libraries which
 aids the development of APIs, client and server, based on
-[GRPC](http://grpc.io) and Google API conventions.
+[gRPC](http://grpc.io) and Google API conventions.
 
 Application code will rarely need to use most of the classes within this
-library directly, but code generated automatically from the API definition
-files can use services such as page streaming and request bundling to provide
+library directly, but code generated automatically from the [API definition
+files](https://github.com/googleapis/googleapis/)
+can use services such as page streaming and request bundling to provide
 a more convenient and idiomatic API surface to callers.
 
+Quickstart
+---
+
+Currently there is no public package for this library. When we come
+closer to release, it will be published on
+[NuGet](https://nuget.org) with an expected package name of
+`Google.Api.Gax`. However, it will also be dependency of specific
```

Comment:
Fixed, thanks. (Will squash, push and merge when I've fixed any other comments.)